apps/ledger-live-desktop/tests/jestSetup.js

```diff
@@ -94,3 +94,18 @@ jest.mock("src/sentry/renderer", () => ({
   setTags: jest.fn(),
   getSentryIfAvailable: jest.fn().mockReturnValue(false),
 }));
+
+if (!globalThis.Buffer) {
+  // Note: this polyfill depends on the patch buffer@6.0.3 which adds the Uint8
+  // subarray logic. It's the same as in ledger-live-mobile
+  // Furthermore, importing 'buffer' gets translated to 'node:buffer' so we're
+  // using a relative path here
+  const { Buffer } = require("../node_modules/buffer");
```

Comment:
Are you sure that using a polyfill does not impact some other coins? Maybe @hedi-edelbloute you have some insights here?